assets/src/domain/eventEditor/ui/tickets/ticketPriceCalculator/fields/MoneyField/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+
+import { MoneyFieldProps } from './types';
+
+import { CurrencySign, PercentSign } from '@appInputs/priceTypeSign';
+
+import useConfig from '@appServices/config/useConfig';
+import './style.scss';
+
+const MoneyField: React.FC<MoneyFieldProps> = ({ children, isPercent = false, ...props }) => {
```

Comment:
because the container has flex display, the currency sign will take needed space w/o the need to set the width explicitly:

![image](https://user-images.githubusercontent.com/1477453/83026917-19d19600-a038-11ea-963c-8b1308f8668d.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/83026989-32da4700-a038-11ea-9586-18c96affec3f.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/83027147-70d76b00-a038-11ea-934b-25b7d7f2f8e9.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/83027341-aed48f00-a038-11ea-92ab-6e1d6579d627.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/83027392-bc8a1480-a038-11ea-9e5e-3d04e6e452cd.png)

Reply:
would be nice to add a bg like this for currency sign: 

![image](https://user-images.githubusercontent.com/1477453/83032427-7b94fe80-a03e-11ea-99d1-51894cc0f8ed.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/83032817-fbbb6400-a03e-11ea-890f-05b48722ea85.png)

Reply:
![image](https://user-images.githubusercontent.com/1477453/83032857-070e8f80-a03f-11ea-88f2-cd0102e74c82.png)

Reply:
i think i like that idea...
the only issue is that it still won't look as nice if the amount inputs in the TPC don't all line up evenly